mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -60,15 +63,19 @@ Feature: Calendar periods
     And param endDate = '2120-08-03'
     When method GET
     Then status 200
-    And match $.totalRecords == 2
```

Comment:
Same as for [:44](https://github.com/folio-org/folio-integration-tests/pull/495/files#diff-58b96cd93d0a7cb94e0cbd76f2e5e4b2458bd1de94cc362bb487158ba3a385d5R44)